src/System/Requirement/SessionsSecurityConfiguration.php

```diff
@@ -49,20 +49,26 @@ public function __construct()
 
     protected function check()
     {
+        $is_cli = isCommandLine();
+
         $cookie_secure   = (bool)ini_get('session.cookie_secure');
         $cookie_httponly = (bool)ini_get('session.cookie_httponly');
         $cookie_samesite = ini_get('session.cookie_samesite');
 
         $is_https_request = ($_SERVER['HTTPS'] ?? 'off') === 'on' || (int)($_SERVER['SERVER_PORT'] ?? null) == 443;
 
-        $validated = true;
-        if ($is_https_request && !$cookie_secure) {
+        if ($is_cli) {
+            $this->validation_messages[] = __('Checking the session cookie configuration of the web server cannot be done in the CLI context.');
+            $this->validation_messages[] = __('However, you should apply the following recommendations for configuring the web server.');
```

Comment:
"However" seems a bit strange to me here (@cconard96 could you please review strings?)

Reply:
The "However," part could just be removed.